lib/puppetserver/ca/errors.rb

```diff
@@ -1,6 +1,20 @@
 module Puppetserver
   module Ca
-    class Error < StandardError; end
+    class Error < StandardError
+      def self.create(ex, msg)
```

Comment:
Are you doing it this way because you wanted to keep the same signature on the `initialize` from `StandardError`?

Reply:
Yeah, and I didn't love the idea of making it a second optional argument to initialize.